GCRCatalogs/catalog_configs/cosmoDC2_v1.1.4_image_combined_agn.yaml

```diff
@@ -0,0 +1,8 @@
+subclass_name: agncombined.AGNCombinedCatalog
```

Comment:
This whole config file can be written as:

```yaml
based_on: cosmoDC2_v1.1.4_image_composite_agn
suppress_overwrite: ['ra', 'redshift', 'is_central', 'halo_mass', 'galaxy_id', 'dec', 'stellar_mass']
```

to avoid repetition.